client/gui/InterfaceObjectConfigurable.cpp

```diff
@@ -92,19 +92,10 @@ std::string InterfaceObjectConfigurable::readText(const JsonNode & config) const
 	
 	const std::string delimiter = "/";
 	std::string s = config.String();
+	boost::replace_all(s, "/", "." );
```

Comment:
Instead, let's change <widget>.json files

Reply:
Agreed.
I believe right now there is only one mod that uses this code - your rmg template selection window.
To reduce disruptions I suggest:
1) wait for merge of this PR
2) change mod .json(s)
3) remove this code in separate PR

Reply:
No worry about mod. We can update mod later, as it non-production anyway

Reply:
Done. Changed cpp code and existing .json from code to match changes in branch:
- for labels, loading is done via ID in form of "core.genrltxt.XXX" where XXX is old string number
- for buttons, loading is done either in form of "core.help.XXX" (in which vcmi will lookup strings "core.help.XXX.help" and "core.help.XXX.hover")
- or, in form of directly specifying both help & hover strings: `"help" : { "help" : "...", "hover" : "..." }`